<launch>
  <node name="tabletop_pushing_perception" pkg="tabletop_pushing" type="tabletop_pushing_perception_node" respawn="false" output="screen">
    <remap from="color_image_topic" to="/kinect_head/rgb/image_color"/>
    <remap from="depth_image_topic" to="/kinect_head/depth/image"/>
    <remap from="point_cloud_topic" to="/kinect_head/depth/points"/>
    <param name="segment_k" value="500.0" type="double"/>
    <param name="segment_sigma" value="0.4" type="double"/>
    <param name="segment_min_size" value="100" type="int"/>
    <param name="segment_color_weight" value="0.4" type="double"/>
    <param name="segment_depth_weight" value="0.7" type="double"/>
    <param name="min_flow_thresh" value="1" type="int"/>
    <param name="num_moving_points_per_region_thresh" value="2" type="int"/>
    <param name="crop_min_x" value="15" type="int"/>
    <param name="crop_max_x" value="600" type="int"/>
    <param name="crop_min_y" value="30" type="int"/>
    <param name="crop_max_y" value="470" type="int"/>
    <param name="display_wait_ms" value="3" type="int"/>
    <param name="min_workspace_x" value="-1.5" type="double"/>
    <param name="max_workspace_x" value="1.5" type="double"/>
    <param name="min_workspace_y" value="-2.0" type="double"/>
    <param name="max_workspace_y" value="2.0" type="double"/>
    <param name="min_workspace_z" value="-1.25" type="double"/>
    <param name="max_workspace_z" value="1.0" type="double"/>
    <param name="workspace_frame" value="/" type="string"/>
  </node>
</launch>
